src/nvim/charset.c

```diff
@@ -348,7 +348,7 @@ char_u *transstr(char_u *s) FUNC_ATTR_NONNULL_RET
         p += l;
 
         if (vim_isprintc(c)) {
-          len += l;
+          len += (size_t)l;
```

Comment:
maybe change the type of `l` to `size_t` (and add casts when setting it if needed)